code/obj/artifacts/artifact_items/talisman.dm

```diff
@@ -0,0 +1,205 @@
+#define STRENGTH 5
+#define PROTECTION 6
+
+/obj/item/artifact/talisman
+	name = "artifact talisman"
+	icon = 'icons/obj/artifacts/artifactsitemS.dmi'
+	associated_datum = /datum/artifact/talisman
+	w_class = W_CLASS_SMALL
+	var/associated_effect
+	var/mob/living/carbon/human/active_user = null
+
+	// swiftness vars
+	var/swiftness_mod = 0
+	// fortune vars
+	var/money_amt = 0
+	// spacefaring vars
+	var/space_prot = 0
+	// elements vars
+	var/heat_prot = 0
+	var/cold_prot = 0
+	// strength vars
+	var/extra_hp = 0
+	// protection vars
+	var/brute_prot = 0
+	var/burn_prot = 0
+	var/tox_prot = 0
+
+	attack_self(mob/user)
+		. = ..()
+		if (!src.artifact.activated)
+			return
+		var/msg
+		switch(src.associated_effect)
+			if (SWIFTNESS)
+				msg = "You feel drafts of air..."
+			if (FORTUNE)
+				msg = "You feel lucky somehow."
+			if (SPACEFARING)
+				msg = "[src] feels ice cold."
+			if (ELEMENTS)
+				msg = "[src] feels warm and cold in different spots.[prob(99) ? null : " Sort of like that honk-pocket you once had..."]"
+			if (STRENGTH)
+				msg = "Holding [src] makes you feel strong."
+			if (PROTECTION)
+				msg = "You feel safe holding [src]."
+
+		boutput(user, SPAN_NOTICE(msg))
+
+	set_loc(newloc, storage_check)
+		..()
+		src.refresh_user()
+
+	parent_storage_loc_changed()
+		src.refresh_user()
+
+	emp_act()
+		if (src.artifact.activated)
+			src.ArtifactDeactivated()
+			playsound(get_turf(src), 'sound/effects/shielddown2.ogg', 75, TRUE)
+
+	proc/refresh_user()
+		if (QDELETED(src) || !src.artifact.activated)
+			return
+
+		var/atom/A = src.get_holder()
+
+		if (src.active_user && A == src.active_user)
+			return
+		if (src.active_user)
+			src.remove_effect_from_user()
+		if (ishuman(A))
+			src.add_effect_to_user(A)
+
+	proc/add_effect_to_user(mob/user)
+		switch(src.associated_effect)
+			if (SWIFTNESS)
+				APPLY_ATOM_PROPERTY(user, PROP_MOB_TALISMAN_SWIFTNESS, src, src.swiftness_mod)
+				APPLY_MOVEMENT_MODIFIER(user, /datum/movement_modifier/artifact_talisman_swiftness, src)
+			if (FORTUNE)
+				user.changeStatus("art_talisman_fortune", null)
+			if (SPACEFARING)
+				APPLY_ATOM_PROPERTY(user, PROP_MOB_COLDPROT, src, src.space_prot)
+			if (ELEMENTS)
+				APPLY_ATOM_PROPERTY(user, PROP_MOB_HEATPROT, src, src.heat_prot)
+				APPLY_ATOM_PROPERTY(user, PROP_MOB_COLDPROT, src, src.cold_prot)
+			if (STRENGTH)
+				user.changeStatus("talisman_extra_hp", null, src.extra_hp)
+			if (PROTECTION)
+				if (src.brute_prot)
+					APPLY_ATOM_PROPERTY(user, PROP_MOB_TALISMAN_BRUTE_REDUCTION, src, src.brute_prot)
+				if (src.burn_prot)
+					APPLY_ATOM_PROPERTY(user, PROP_MOB_TALISMAN_BURN_REDUCTION, src, src.burn_prot)
+				if (src.tox_prot)
+					APPLY_ATOM_PROPERTY(user, PROP_MOB_TALISMAN_TOX_REDUCTION, src, src.tox_prot)
+
+		src.active_user = user
+		user.setStatus("art_talisman_held", null, src)
+
+	proc/remove_effect_from_user()
+		if (!src.active_user)
+			return
+
+		src.active_user.delStatus("art_talisman_held")
```

Comment:
how does this interact with having multiple talismans active?

Reply:
Yeah wasn't working correctly, should be fixed now